CHANGES.rst

```diff
@@ -4,6 +4,8 @@
 New Features
 ------------
 
+- Plots in plugins now include basic zoom/pan tools. [#2498]
```

Comment:
```suggestion
- Plots in plugins now include basic zoom/pan tools for Plot Options,
  Imviz Line Profiles, and Imviz Simple Aperture Photometry. [#2498]
```

Reply:
is this just because of concern it would otherwise suggest the plugin?  (I don't mind being more explicit, but #2465 is also changing the name of the aperture photometry plugin and so we'd have to remember there to update this as well).